### ISSUE TITLE
Ultimaker 2 is extruding poorly

### ISSUE THREAD

**Issue by amcewen** [DoESLiverpool/somebody-should#393]
The standard Ultimaker 2 isn't extruding very well at the moment.  It will extrude, but the extruded filament is thinner than it should be.  I improve matters a bit on the last print I did by pushing the temperature up to 215 and printing in "ulti" quality (which doesn't need the extruder to push as much plastic out).

I guess first step - if no-one has tried it recently - would be to clean the nozzle out with the cold pull method.  If that doesn't improve matters then it's probably worth checking the state of the PTFE coupling.

**Comment by DefProc** [comment]
Julian mentioned that he'd dine a few cold pulls recently, with little
improvement.

**Comment by amcewen** [comment]
Looked at this a tiny bit yesterday, and it takes **lots** of pressure to manually force any plastic through the extruder, so I'm moving it up to broken.

**Comment by goatchurchprime** [comment]
I thought about exchanging the nozzles to get at least one printer going well, but got as far as dismantling only the nozzle on the ultimaker before deciding not to completely wreck both of them.  (The other still prints, just with bad tolerance.)

Maybe we should exchange the rods if we want to get one into better order fastest.  

There was a spare used nozzle in the box, but it looked as bad as the one that's there.  

I can't get the temperature sensor out of the nozzle block and I don't want to break it.  

Anyway, we really ought to have some running spares for this sort of thing as these machines are getting a lot of use.

**Comment by DefProc** [comment]
Also probably worth getting the upgraded heater head, so we can clean and
change nozzles without interfering with the heater block

**Comment by goatchurchprime** [comment]
Which spare part(s) are you referring to?  
https://ultimaker.com/en/products/spare-parts
https://www.dream3d.co.uk/product-category/upgrades-spare-parts/

(I don't see any sales of new rods in those sites.)

Maybe we should keep an update of spare parts supplies and sources on this page:
    https://github.com/DoESLiverpool/wiki/wiki/Ultimaker

**Comment by amcewen** [comment]
I think we should get this back up and running first rather than decide to leave it broken and embark upon a massive upgrade of the extruder - @DefProc, feel free to spin the upgrade out as a new issue.

@DoESsean has bought a new PTFE coupling (as I still haven't found the spares mentioned in #283).  The old one is nicely blocked with some filament, so it's possible that just replacing that (and doing a cold pull to clean out the well burnt bits of PLA left in the nozzle) might get it up and running again.

I had a go at removing the nozzle during makernight last week, but didn't have any more joy than @goatchurchprime.  I've left the new PTFE coupling with the printer, so I think the next step would be reassembling it with the new coupling and seeing if that gets us back up and running.

**Comment by amcewen** [comment]
@goatchurchprime I've made a start on a spares section on the wiki https://github.com/DoESLiverpool/wiki/wiki/Ultimaker#spares

**Comment by DefProc** [comment]
The olsen block is apparently a good update for the UM2: 
https://www.dream3d.co.uk/product/ultimaker-2-olsson-block/

**Comment by JimNixon** [comment]
I've got an Ohlson block in mine and its great. Makes future nozzle
replacements a breeze.

**Comment by amcewen** [comment]
@ajlennon and I reassembled the extruder with the new PTFE coupler, and with some cold pulls, fiddling with needles, etc. managed to get the nozzle unblocked and plastic flowing.

However, while doing all that there was a build-up of PLA in between the bottom of the PTFE coupler and the nozzle (presumably forced there when there wasn't a clear path through the nozzle and so nowhere else for the plastic to go) which blocked the PTFE coupler.

So, disassembled and cleaned things out again - @jackie1050 suggested a great trick of using a lighter to warm the PLA plugging the coupler so it softened and could be pulled out.

All reassembled now and printing really nicely.

**Comment by DefProc** [comment]
The ultimaker has stopped extruding again.

I've done a few cold pulls (with @goatchurchprime) and there seems to be a blockage in the tip that won't attach to the hot filament. 

I've ordered some 0.4mm stainless wire to be able to push upwards to see of that will clear it, but of it needs stripping down, then it might be worth considering an Olsson Block replacement at the same time.

**Comment by DefProc** [comment]
The ultimaker has stopped extruding again.

I've done a few cold pulls (with @goatchurchprime) and there seems to be a blockage in the tip that won't attach to the hot filament. 

I've ordered some 0.4mm stainless wire to be able to push upwards to see of that will clear it, but of it needs stripping down, then it might be worth considering an Olsson Block replacement at the same time.

**Comment by DefProc** [comment]
There's an ultimaker page listing the steps for clearing a nozzle now at: https://ultimaker.com/en/resources/21619-how-to-unclog-an-ultimaker-nozzle

**Comment by JimNixon** [comment]
I'll be down Thursday night so can check it out.

**Comment by JimNixon** [comment]
I'll be down Thursday night so can check it out.

On 17 Jan 2017 16:20, "Patrick Fenner" <notifications@github.com> wrote:

There's an ultimaker page listing the steps for clearing a nozzle now at:
https://ultimaker.com/en/resources/21619-how-to-unclog-an-ultimaker-nozzle

—
You are receiving this because you commented.

Reply to this email directly, view it on GitHub
<https://github.com/DoESLiverpool/somebody-should/issues/393#issuecomment-273217236>,
or mute the thread

**Comment by DefProc** [comment]
We've got some 0.4mm stainless steel wire to help with cleaning the nozzle. I'll leave it on the UM2. It's marked: `3D printer nozzle cleaning`

**Comment by JimNixon** [comment]
Had a look at it and the PTFE coupler is worn out. Chatted to Adrian about it who said that it was replaced recently, so I suspect it's a counterfeit part. I've been using the ones manufactured by 3dSolex (the creator of the Olhussen block upgrade) and they're better than even the proper ones from Ultimaker. Ordered from their UK distributor before and had great service.

http://www.goodtube.uk/product-page/21dadf0e-08e3-ba79-32fa-69c3a20f9f79

**Comment by DoESsean** [comment]
Jim's recommendation was out of stock, so we've ordered one from [https://www.dream3d.co.uk/product/ultimaker-2-ptfe-coupler/](url)

**Comment by amcewen** [comment]
@JimNixon fitted the new coupler at Maker Night and it seems to be back up and printing nicely (currently printing off a new nozzle for the vacuum cleaner to fix #436)
